### DISCUSSION TITLE
Set dark mode for one component

### DISCUSSION THREAD

**Discussion by rtorcato** [mantinedev/mantine#1202] [Q&A] [answered]
I know I can use style prop, but was wondering if there was a quick way toggle a component to use dark mode  without using dark mode on the entire site?

**Comment by rtivital** [comment, marked as answer]
Use nested MantineProvider – https://mantine.dev/theming/mantine-provider/#nested-mantineproviders

**Reply by rtorcato** [reply]
great thanks

**Reply by xe4me** [reply]
<img width="877" height="188" alt="Screenshot 2025-10-10 at 00 54 35" src="https://github.com/user-attachments/assets/ae81d6ba-7b21-465b-9c73-00ec38c64638" />

It clearly says it cannot be nested

**Reply by rtivital** [reply]
It is no longer supported starting from version 7.0.

**Reply by xe4me** [reply]
Thank you @rtivital 
So there's no way to achieve this ? 
Problem is, say in a light mode, if I want to put a black background under a section, I'd have to manually set color of every text element to be light, which is easy enough, but it gets more complicated with things like button that have more complex coloring.
Where as if this was supported, I would easily the the theme as dark, and everything would just work

**Reply by rtivital** [reply]
Yes, I could not make this work, there are issues with CSS variables specificity. To not override all text colors, use

```css
.darkSection * {
  color: white !important;
}
```
